docs/tre-admins/auth.md

```diff
@@ -192,7 +192,9 @@ Same as [TRE API](#authentication-tre-api).
 | API/permission name | Type | Description | Admin consent required |
 | ------------------- | ---- | ----------- | ---------------------- |
 | Microsoft Graph/User.Read (`https://graph.microsoft.com/User.Read`) | Delegated | Allows users to sign-in to the app, and allows the app to read the profile of signed-in users. It also allows the app to read basic company information of signed-in users. | No |
-| Workspace API/user_impersonation (`api://<Workspace API Application (client) ID>/user_impersonation`) | Delegated* | Allows the app to access the workspace API on behalf of the user | No | Granted for *[directory name]* |
+| Workspace API/user_impersonation (`api://<Workspace Scope ID>/user_impersonation`) | Delegated* | Allows the app to access the workspace API on behalf of the user | No | Granted for *[directory name]* |
+
+When the Workpsace AAD app is registered by the aad-app-reg.sh, the `Workspace Scope Id` is the same as the Client Id.  
```

Comment:
Could you add a sentence saying something like "Otherwise, when the Workspace AAD app is registered by the bundle, the `Workspace Scope Id` is defined as ....."

Reply:
Also, there's a typo here (`Workpsace` should be `Workspace`)